sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/ConfigurationException.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.autoconfigure;
+
+/** An exception that is thrown if the user-provided configuration is invalid. */
+public final class ConfigurationException extends RuntimeException {
+
+  private static final long serialVersionUID = 4717640118051490483L;
```

Comment:
I don't think this is necessary or useful. No one should be using default serialization for anything, anywhere.

Reply:
I think Java throws a warning which fails our build if I don't have it (agree no one should serialize it :)

warning: [serial] serializable class ConfigurationException has no definition of serialVersionUID

Reply:
that must be an errorprone warning. I can't imagine that javac would emit that.